rotkehlchen/tests/unit/decoders/test_gnosis_bridge.py

```diff
@@ -0,0 +1,153 @@
+import pytest
+
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.accounting.structures.evm_event import EvmEvent
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.ethereum.modules.xdai_bridge.decoder import BRIDGE_ADDRESS
+from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
+from rotkehlchen.chain.gnosis.constants import CPT_XDAI_BRIDGE
+from rotkehlchen.chain.gnosis.modules.xdai_bridge.decoder import (
+    BRIDGE_ADDRESS as GNOSIS_BRIDGE_ADDRESS,
+)
+from rotkehlchen.constants.assets import A_DAI, A_ETH, A_XDAI
+from rotkehlchen.fval import FVal
+from rotkehlchen.tests.utils.ethereum import get_decoded_events_of_transaction
+from rotkehlchen.types import Location, TimestampMS, deserialize_evm_tx_hash
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0x4c1a316De360E08817eB88dD31A0E7305005fB65']])
+def test_bridge_dai_from_ethereum(
+        database,
+        ethereum_inquirer,
+        ethereum_accounts,
+):
```

Comment:
this fits in one line. same for the other functions

```suggestion
def test_bridge_dai_from_ethereum(database, ethereum_inquirer, ethereum_accounts):
```